src/components/RecentTickets.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import Moment from 'react-moment';
+
+const recentTickets = {
+  display: 'inline-block',
+  marginTop: '20px',
+};
+
+export default class RecentTickets extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { lastResponse: null, lastError: null };
+  }
+
+  render() {
+    // TODO: pagination
+    return (
+      <div style={recentTickets}>
+        {!this.state.lastResponse && !this.state.lastError ? (
+          <div>Loading recent tickets...</div>
+        ) : null}
+
+        {this.state.lastError ? (
+          <div>Failed to load recent tickets: {this.state.lastError}</div>
+        ) : null}
+
+        {this.state.lastResponse ? (
+          <div>
+            <h3>Recent Tickets:</h3>
+            <table className="ui celled padded table">
+              <thead>
+                <tr>
+                  <th>URL</th>
+                  <th>Submitted</th>
+                  <th>Processed</th>
+                  <th>Output</th>
+                </tr>
+              </thead>
+              <tbody>
+                {this.state.lastResponse.map(ticket => (
+                  <tr>
+                    <td>{ticket.url}</td>
+                    <td>
+                      <Moment date={ticket.CreatedAt} fromNow />
+                    </td>
+                    <td>{ticket.processed ? 'Yes' : 'No'}</td>
+                    <td>
+                      <a href={`/ticket/${ticket.ID}`}>Ticket #{ticket.ID}</a>
+                    </td>
+                  </tr>
+                ))}
+              </tbody>
+              <tfoot>
+                <tr>
+                  <th colSpan="5">
+                    <div className="ui right floated pagination menu">
+                      <a className="icon item">
+                        <i className="left chevron icon"></i>
+                      </a>
+                      <a className="item">1</a>
+                      <a className="item">2</a>
+                      <a className="item">3</a>
+                      <a className="item">4</a>
+                      <a className="icon item">
+                        <i className="right chevron icon"></i>
+                      </a>
+                    </div>
+                  </th>
+                </tr>
+              </tfoot>
```

Comment:
This is great William!! The only comment I have is that we should modularize this into separate components (for example a TableRow component, pagination component), and prefer importing components directly from semantic-ui-react over using native html.